src/components/mode-tools/mode-tools.jsx

```diff
@@ -33,185 +37,446 @@ import straightPointIcon from './icons/straight-point.svg';
+                                    onMouseOver={this.handleChangeFontSerif}
+                                >
+                                    <span className={styles.serif}>
+                                        {this.props.intl.formatMessage(messages.serif)}
+                                    </span>
+                                </Button>
+                                <Button
+                                    className={classNames(styles.modMenuItem)}
+                                    onClick={this.handleClick}
+                                    onMouseOver={this.handleChangeFontHandwriting}
+                                >
+                                    <span className={styles.handwriting}>
+                                        {this.props.intl.formatMessage(messages.handwriting)}
+                                    </span>
+                                </Button>
+                                <Button
+                                    className={classNames(styles.modMenuItem)}
+                                    onClick={this.handleClick}
+                                    onMouseOver={this.handleChangeFontMarker}
+                                >
+                                    <span className={styles.marker}>
+                                        {this.props.intl.formatMessage(messages.marker)}
+                                    </span>
+                                </Button>
+                                <Button
+                                    className={classNames(styles.modMenuItem)}
+                                    onClick={this.handleClick}
+                                    onMouseOver={this.handleChangeFontCurly}
+                                >
+                                    <span className={styles.curly}>
+                                        {this.props.intl.formatMessage(messages.curly)}
+                                    </span>
+                                </Button>
+                                <Button
+                                    className={classNames(styles.modMenuItem)}
+                                    onClick={this.handleClick}
+                                    onMouseOver={this.handleChangeFontPixel}
+                                >
+                                    <span className={styles.pixel}>
+                                        {this.props.intl.formatMessage(messages.pixel)}
+                                    </span>
+                                </Button>
+                                <Button
+                                    className={classNames(styles.modMenuItem)}
+                                    onClick={this.handleClick}
+                                    onMouseOver={this.handleChangeFontChinese}
+                                >
+                                    <span className={styles.chinese}>
+                                        中文
```

Comment:
Is there something else we can show that indicates that this font supports Chinese (ditto for Korean and Japanese)? It was definitely my first question opening the font menu, as I can't identify characters belonging to those different languages. I think this will cause some consternation among kids (although it also piques curiosity!) Maybe just writing `中文 (Chinese)` would help? If nobody else thinks it is a big deal then nevermind. 

/cc @carljbowman

Reply:
Our plan was to not translate these 3 font names. Using 2 languages might make translation tricky. We could use the name of the language and translate that, but @thisandagain and @carljbowman  liked this better I think.

Reply:
That reminds me, the font names try to be in the font style, so if the font names are translated to unsupported characters, they might look weird. For instance you can see the effect of Marker's limited character set here:
![image](https://user-images.githubusercontent.com/2855464/40142735-821814a0-5927-11e8-82e1-191a97516c95.png)

Ideas for how to handle this? It mostly looks broken for languages where the translation includes the occasional character with an accent.
@chrisgarrity

Reply:
We decided to resolve this by not translating any font names